README.rst

```diff
@@ -3129,7 +3190,7 @@ a specfic connection object for the remote retrieval, you can set it
 as second parameter::
 
      require 'vendor/autoload.php';
-     $api = new BigML(my_username, my_api_key);
+     $api = new BigML();
```

Comment:
If you want to use the deepnet locally, it would make sense to add the storage parameter to the api connection. Otherwise, the JSON retrieved is not stored locally, so you would not really have a local deepnet.

Reply:
In my understanding, local deepnet is a way to get the predictions calculated locally. The data is held in memory as long as the variable bound to it is not collected. If you need to add some form of persistency to this local behaviour, you add a storage. So, not sure we should suggest that local deepnet is only when you use a storage -- it depends on what you are doing.

Reply:
Your understanding is right, in the sense that the prediction computation is done locally either way. However, it is usually the case that users want the Deepnet object to be as local as possible, so that once the JSON has been downloaded once, you don't need to connect ever again to the internet. If we provide examples that have this kind of behaviour we will be encouraging it and will receive less queries in support asking why the Deepnet object takes so long to isntantiate/predict, so I think it would be wise to add the storage argument to the api explicit connection here.

Reply:
I'm sorry, but I don't agree. :) This is what it's done implicitly inside of the Deepnet local object
```

        if ($api == null) {
            $api = new BigML(null, null, null, $storage);
        }
```

and the text is claiming that you can do that explicitly and shows the example. That example should have the storage argument. You can also explain that this storage can be removed and then the Deepnet would connect to download the JSON every time is instantiated, that I can agree.